starlette/middleware/wsgi.py

```diff
@@ -43,7 +43,7 @@ def build_environ(scope: Scope, body: bytes) -> dict:
         elif name == "content-type":
             corrected_name = "CONTENT_TYPE"
         else:
-            corrected_name = "HTTP_%s" % name.upper().replace("-", "_")
+            corrected_name = f"HTTP_{name.upper().replace("-", "_")}"
```

Comment:
This isn't valid syntax.  (See test failure... https://travis-ci.org/encode/starlette/jobs/482660913)